daemon/main.cpp

```diff
@@ -588,7 +592,11 @@ win32_daemon_entry(DWORD largc, LPTSTR* largv)
   SvcStatus.dwServiceSpecificExitCode = 0;
 
   // Report initial status to the SCM
-  ReportSvcStatus(SERVICE_RUNNING, NO_ERROR, 0);
-  lokinet_main(largc, largv);
+  ReportSvcStatus(SERVICE_START_PENDING, NO_ERROR, 3000);
+  // SCM clobbers startup args, regenerate them here
+  argc = 2;
+  argv[1] = "c:/programdata/.lokinet/lokinet.ini";
```

Comment:
.....................because we eat it here: becomes `[path to lokinet] [path to config file]`
TODO: for internal testing use `$ALLUSERSPROFILE/Appplication Data/.lokinet` (programdata didn't exist yet)